clients/include/testing_trmm_strided_batched.hpp

```diff
@@ -136,6 +132,66 @@ hipblasStatus_t testing_trmm_strided_batched(const Arguments& argus)
         {
             unit_check_general<T>(M, N, batch_count, ldb, stride_B, hB, hB_copy);
         }
+        hipblas_error = norm_check_general<T>(
+            'F', M, N, ldb, stride_B, hB_copy.data(), hB.data(), batch_count);
```

Comment:
Minor question so does -v sets both unit and arge?   If unit_check is sometimes without norm_check would you want to if (argus.norm_check) to avoid this for test runs?

Reply:
I'm pretty sure unit_check is only used for tests, and norm_check is only used for bench. I'll check tomorrow, if that's the case I'll add an if statement.